x/ibc/02-client/types/tendermint/header.go

```diff
@@ -0,0 +1,48 @@
+package tendermint
+
+import (
+	tmtypes "github.com/tendermint/tendermint/types"
+
+	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/exported"
+)
+
+var _ exported.Header = Header{}
+
+// Header defines the Tendermint consensus Header
+type Header struct {
+	// TODO: define Tendermint header type manually, don't use tmtypes
```

Comment:
Why would we want to do this? To keep the implementations separate?

Reply:
idk. @mossid left this `TODO` here